src/plugins/data/public/ui/query_string_input/query_string_input.tsx

```diff
@@ -379,14 +379,12 @@ export default class QueryStringInputUI extends Component<Props, State> {
     const newQueryString = value.substr(0, start) + text + value.substr(end);
 
     this.reportUiCounter?.(
-      METRIC_TYPE.LOADED,
-      `query_string:${type}:suggestions_select_position`,
-      listIndex
+      METRIC_TYPE.CLICK,
+      `query_string:${type}:suggestions_select_position_${listIndex}`
```

Comment:
Isn't there a way to report this without losing the numeric nature of the data?

Reply:
😕  Not using UI metrics. If we wanted to introduce our own mechanism for collecting/reporting data, we could probably get around it, but I'm not sure it's worth the effort. We can still do side-by-side comparisons of the different autocomplete implementations, we just won't have a single metric to compare side-by-side. (Although we may be able to get around this by using a scripted field...)